composer.json

```diff
@@ -48,7 +48,7 @@
     },
     "autoload-dev": {
         "psr-4": {
-            "Yiisoft\\Yii\\Db\\Migration\\Tests\\": "tests/src",
+            "Yiisoft\\Yii\\Db\\Migration\\Tests\\": "tests",
```

Comment:
We can keep only this namespace and remove `Yiisoft\\Yii\\Db\\Migration\\TestsRuntime\\`.

Reply:
done.